cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
@hferentschik @gbraad I think you both agreed to the point that we need to support the global config, which does not get removed after minishift delete. As a minishift user  I would be like to do minishift start -> stop -> start -> stop -> start and I think we all agree that we need to support this use case anyway and having global config for hostfolder is aligned in the same line of user experience. Let me know if I  misinterpreted it.